pkg/mount/block/blockdev_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package block
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Isn't this very counter to the idiom of the rest of the Go tests where if-statements are used to check test results? Any good reason to bring this in?

Reply:
Yeah it is -- but we accepted it when systemboot came in. I just moved this from pkg/storage.